### ISSUE TITLE
redmine_acts_as_taggable_on (at acts_as_taggable_on-3.5.0) is not yet checked out. Run `bundle install` first.

### ISSUE THREAD

**Issue by min2sia** [alexbevi/redmine_knowledgebase#296]
Hi, All,

I'm trying out the "redmine-3.0-compat" branch with Redmine 3.1.1:

```
$ bundle install
Updating https://github.com/y-yoshinoya/redmine_acts_as_taggable_on
...
Using redmine_acts_as_taggable_on 1.1.0 (was 1.0.0) from https://github.com/y-yoshinoya/redmine_acts_as_taggable_on (at acts_as_taggable_on-3.5.0)
...
```

then 

```
# bundle exec rake redmine:plugins:migrate RAILS_ENV=production
https://github.com/y-yoshinoya/redmine_acts_as_taggable_on (at acts_as_taggable_on-3.5.0) is not yet checked out. Run `bundle install` first.
```

Doing bundle install again shows that redmine_acts_as_taggable_on is already installed:

```
...
Using redmine_acts_as_taggable_on 1.1.0 (was 1.0.0) from https://github.com/y-yoshinoya/redmine_acts_as_taggable_on (at acts_as_taggable_on-3.5.0)
...
Your bundle is complete!
```

Am I missing something here?..

**Comment by thinknot** [comment]
This may be a longshot, but you might want to check out your Gemfile.lock 
I ran into this error message on a debian box where this file was symlinked to /dev/null
Someting to do with the git-based dependency... The fix for me was to avoid apt-get like the plague, and install redmine from source tarball like a normal person.

**Comment by alexbevi** [comment]
@min2sia: did @thinknot's solution help? I just released a new version of the plugin, but i'm not sure if it would resolve this issue

**Comment by Silverjack** [comment]
run into the same problem
![image](https://cloud.githubusercontent.com/assets/761854/12159166/01fcb648-b51b-11e5-8cfb-9d9d51cf280f.png)

**Comment by hello2jack** [comment]
When i use Ruby 2.2.3 & passenger-5.0.26
I cannot run bundle install normally

**Error message:**
`git://github.com/alexbevi/redmine_acts_as_taggable_on.git (at master@a939bae) is not yet checked out. Run `bundle install` first. (Bundler::GitError)`

**Bundle install error message:**
`Fetching git://github.com/alexbevi/redmine_acts_as_taggable_on.git
fatal: unable to connect to github.com:
github.com[0: 192.30.252.128]: errno=Connection timed out

Retrying git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git' "/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979" --bare --no-hardlinks --quiet due to error (2/4): Bundler::Source::Git::GitCommandError Git error: command `git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git' "/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979" --bare --no-hardlinks --quiet` in directory /var/www/redmine has failed.
fatal: unable to connect to github.com:
github.com[0: 192.30.252.131]: errno=Connection timed out

Retrying git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git' "/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979" --bare --no-hardlinks --quiet due to error (3/4): Bundler::Source::Git::GitCommandError Git error: command `git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git' "/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979" --bare --no-hardlinks --quiet` in directory /var/www/redmine has failed.
fatal: unable to connect to github.com:
github.com[0: 192.30.252.131]: errno=Connection timed out

Retrying git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git' "/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979" --bare --no-hardlinks --quiet due to error (4/4): Bundler::Source::Git::GitCommandError Git error: command `git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git' "/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979" --bare --no-hardlinks --quiet` in directory /var/www/redmine has failed.
fatal: unable to connect to github.com:
github.com[0: 192.30.252.130]: errno=Connection timed out

Git error: command `git clone 'git://github.com/alexbevi/redmine_acts_as_taggable_on.git'
"/usr/local/rvm/gems/ruby-2.2.3/cache/bundler/git/redmine_acts_as_taggable_on-82f0b3fd11b92d791c4838695491decb70c3b979"
--bare --no-hardlinks --quiet` in directory /var/www/redmine has failed.`

**Comment by mathiaswallgren** [comment]
I also have this problem and I am not very good with Ruby. I already had the CRM plugin (which I am not using) and redmine_knowledgebase would not work. I tried to uninstall the CRM plugin which failed and I tried to uninstall this plugin which also failed. Now my the nginx server will not start and my Redmine site is down and I have tried to find a solution and spent several hours on this because I have a lot of data in Redmine and use it every day. The error I get when I try an uninstall is that the source for redmine_acts_as_taggable is not yet checked out. 

![uninstall_error](https://cloud.githubusercontent.com/assets/15824187/24613047/71e729ac-18ca-11e7-8ce1-4ab6ce5913c4.JPG)

If I try a bundle install I get a message saying that the version for this gem in the gemfile is ~>4.0 and redmine_acts_as_taggable was resolved to 1.1.0, which depends ~>3.4. Any suggestions how I can resolve this? I'd really would like this plugin, but now I just need to get Redmine working again.

![error](https://cloud.githubusercontent.com/assets/15824187/24612953/209c44d8-18ca-11e7-88e3-da5720051f8b.JPG)

Is it a way for me to just remove both of these plugins and get Redmine running again, but without affecting my other plugins?
